From 0b225005e614de404ae59730f2cb7af8d4093184 Mon Sep 17 00:00:00 2001
From: Judas Drekonym <77771385+RAMDRAGONS@users.noreply.github.com>
Date: Wed, 8 Jul 2026 01:21:49 -0400
Subject: [PATCH] sched/fair: revert cachy mods

Signed-off-by: CachyOS <admin@cachyos.org>
---
 kernel/sched/fair.c | 10 +++++++---
 1 file changed, 7 insertions(+), 3 deletions(-)

diff --git a/kernel/sched/fair.c b/kernel/sched/fair.c
index 40d95c1e55..7f5613f98f 100644
--- a/kernel/sched/fair.c
+++ b/kernel/sched/fair.c
@@ -75,7 +75,7 @@
  *
  * (default SCHED_TUNABLESCALING_LOG = *(1+ilog(ncpus))
  */
-unsigned int sysctl_sched_tunable_scaling = SCHED_TUNABLESCALING_NONE;
+unsigned int sysctl_sched_tunable_scaling = SCHED_TUNABLESCALING_LOG;
 
 /*
  * Minimal preemption granularity for CPU-bound tasks:
@@ -86,11 +86,15 @@ unsigned int sysctl_sched_tunable_scaling = SCHED_TUNABLESCALING_NONE;
 unsigned int sysctl_sched_base_slice			= 400000ULL;
 static unsigned int normalized_sysctl_sched_base_slice	= 400000ULL;
 #else
-unsigned int sysctl_sched_base_slice			= 800000000ULL / HZ * (HZ > 500? 2: 1);
-static unsigned int normalized_sysctl_sched_base_slice	= 800000000ULL / HZ * (HZ > 500? 2: 1);
+unsigned int sysctl_sched_base_slice			= 700000ULL;
+static unsigned int normalized_sysctl_sched_base_slice	= 700000ULL;
 #endif /* CONFIG_CACHY */
 
+#ifdef CONFIG_CACHY
 __read_mostly unsigned int sysctl_sched_migration_cost	= 400000UL;
+#else
+__read_mostly unsigned int sysctl_sched_migration_cost	= 500000UL;
+#endif
 
 static int __init setup_sched_thermal_decay_shift(char *str)
 {
-- 
2.55.0

