diff --git i/kernel/sched/fair.c w/kernel/sched/fair.c
index 4929fdf049..407d409ac0 100644
--- i/kernel/sched/fair.c
+++ w/kernel/sched/fair.c
@@ -75,7 +75,7 @@
  * BORE : default SCHED_TUNABLESCALING_NONE = *1 constant
  * EEVDF: default SCHED_TUNABLESCALING_LOG  = *(1+ilog(ncpus))
  */
-unsigned int sysctl_sched_tunable_scaling = SCHED_TUNABLESCALING_NONE;
+unsigned int sysctl_sched_tunable_scaling = SCHED_TUNABLESCALING_LOG;
 
 /*
  * Minimal preemption granularity for CPU-bound tasks:
@@ -89,11 +89,11 @@ static const unsigned int nsecs_per_tick       = 1000000000ULL / HZ;
 unsigned int sysctl_sched_min_base_slice       = CONFIG_MIN_BASE_SLICE_NS;
 __read_mostly uint sysctl_sched_base_slice     = nsecs_per_tick;
 #else /* !CONFIG_SCHED_BORE */
-unsigned int sysctl_sched_base_slice			= 800000000ULL / HZ * (HZ > 500? 2: 1);
-static unsigned int normalized_sysctl_sched_base_slice	= 800000000ULL / HZ * (HZ > 500? 2: 1);
+unsigned int sysctl_sched_base_slice			= 700000ULL;
+static unsigned int normalized_sysctl_sched_base_slice	= 700000ULL;
 #endif /* CONFIG_SCHED_BORE */
 
-__read_mostly unsigned int sysctl_sched_migration_cost	= 400000UL;
+__read_mostly unsigned int sysctl_sched_migration_cost	= 500000UL;
 
 static int __init setup_sched_thermal_decay_shift(char *str)
 {
